packages/whenable/README.md

```diff
@@ -0,0 +1,69 @@
+# Whenables
```

Comment:
Is this expected to be part of the developer API product?

If so, I'd like to see generated tsdocs.